SEED-platform/seed
# fix filename displays in dataset/mapping screen

#### Any background context you want to provide?
The names displayed for the files were the tmp filename that were generated.

#### What's this PR do?
Fixes the display names to be the uploaded filename without the unique string generation.

#### How should this be manually tested?
Monkey testing by importing new files. Make sure to check existing files as well. For older files, the display name will not change (i.e. the filename with the random string appended).

#### What are the relevant tickets?
#1184 

#### Screenshots (if appropriate)
#### Definition of Done:
**Make sure to run migrate**
